src/java.base/share/classes/module-info.java

```diff
@@ -278,7 +278,8 @@
         jdk.net;
     exports sun.net to
         java.net.http,
-        jdk.naming.dns;
+        jdk.naming.dns,
+        jdk.httpserver;
```

Comment:
It would be good if we could avoid giving the jdk.httpserver module access to the sun.net.* package.

@dfuch @Michael-Mc-Mahon Are you sure that net.properties is the right place for this?

Reply:
I was surprised that the HttpServer didn't already have access to `sun.net.*` - and I agree that it would be better if we did not add such an access. That said - the new property is a compatibility property - and documenting in module-info seemed a bit too much - so out of the way in `net.properties` seemed like an appealing place to add it. 

Another possibility could be to document it only in the release notes.